crates/summit/src/grpc.rs

```diff
@@ -235,6 +247,46 @@ async fn refresh(state: Arc<State>, request: tonic::Request<()>) -> Result<(), E
     Ok(())
 }
 
+#[tracing::instrument(skip_all)]
+async fn pause(state: Arc<State>, request: tonic::Request<()>) -> Result<(), Error> {
+    let token = request
+        .extensions()
+        .get::<VerifiedToken>()
+        .cloned()
+        .ok_or(Error::MissingRequestToken)?;
+
+    let account_id = token.decoded.payload.account_id;
+
+    info!(
+        account = %account_id,
+        "Pause"
+    );
+
+    let _ = state.worker.send(worker::Message::Pause);
+
+    Ok(())
+}
+
+#[tracing::instrument(skip_all)]
+async fn resume(state: Arc<State>, request: tonic::Request<()>) -> Result<(), Error> {
+    let token = request
+        .extensions()
+        .get::<VerifiedToken>()
+        .cloned()
+        .ok_or(Error::MissingRequestToken)?;
+
+    let account_id = token.decoded.payload.account_id;
+
+    info!(
+        account = %account_id,
+        "Resume"
+    );
```

Comment:
Why not include the `_id` in the tracing field name and shorten this?


```suggestion
    info!(%account_id, "Resume");
```

Reply:
For sure. This is done very inconsistently currently and we need to revisit all spans and logged values to ensure we're using the same convention and also adding human readable values where it makes sense.

For example we log the task ids but it'd be much better if we resolve the associated build_id as soon as possible and add that to the tracing span so it colors all future spans with that info. build_id is very human readable and will make logs easier to follow.